### PR TITLE
Refactroing TypeName tests

### PR DESCRIPTION
**Pull request by frootloops**
I have only one question how can I avoid using `main` and use something like `[[THIS]]` for \* StdlibUnittest*?

### CODE REVIEW

File: test/1_stdlib/TypeName.swift [hunk 1]
```diff
@@ -1,6 +1,11 @@
-// RUN: %target-run-simple-swift | FileCheck %s
+// RUN: %target-run-simple-swift
 // REQUIRES: executable_test
 
+import Foundation
```
Comment (gribozavr) [inline test/1_stdlib/TypeName.swift]: Why do we need Foundation?  (It wasn't imported before.)

**Comment by gribozavr** [comment]
> I have only one question how can I avoid using `main` and use something like `[[THIS]]` for * StdlibUnittest*?

You can define a string in the global scope and use it in string interpolation everywhere, but I think it is perfectly fine to hardcode `main`.

**Comment by gribozavr** [comment]
LGTM modulo the Foundation comment.

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by frootloops** [comment]
@gribozavr ok, thanks

**Comment by gribozavr** [comment]
@swift-ci Please test and merge

**Pull request merged**